vm/actor/src/util/chaos/mod.rs

```diff
@@ -50,24 +76,24 @@ impl Actor {
     ///
     ///  CALLER_VALIDATION_BRANCH_NONE performs no validation.
     ///  CALLER_VALIDATION_BRANCH_TWICE validates twice.
-    ///  CALLER_VALIDATION_BRANCH_ADDR_NIL_SET validates against an empty caller
+    ///  CALLER_VALIDATION_BRANCH_IS_ADDRESS validates against an empty caller
     ///  address set.
-    ///  CALLER_VALIDATION_BRANCH_TYPE_NIL_SET validates against an empty caller type set.
-    pub fn caller_validation<BS, RT>(rt: &mut RT, branch: BigInt) -> Result<(), ActorError>
+    ///  CALLER_VALIDATION_BRANCH_IS_TYPE validates against an empty caller type set.
+    pub fn caller_validation<BS, RT>(rt: &mut RT, branch: i64) -> Result<(), ActorError>
     where
         BS: BlockStore,
         RT: Runtime<BS>,
     {
         match branch {
-            x if x == *CALLER_VALIDATION_BRANCH_NONE => {}
-            x if x == *CALLER_VALIDATION_BRANCH_TWICE => {
+            x if x == CALLER_VALIDATION_BRANCH_NONE => {}
+            x if x == CALLER_VALIDATION_BRANCH_TWICE => {
                 rt.validate_immediate_caller_accept_any()?;
                 rt.validate_immediate_caller_accept_any()?;
             }
-            x if x == *CALLER_VALIDATION_BRANCH_ADDR_NIL_SET => {
+            x if x == CALLER_VALIDATION_BRANCH_IS_ADDRESS => {
                 rt.validate_immediate_caller_is(&[])?;
             }
-            x if x == *CALLER_VALIDATION_BRANCH_TYPE_NIL_SET => {
+            x if x == CALLER_VALIDATION_BRANCH_IS_TYPE => {
                 rt.validate_immediate_caller_type(&[])?;
```

Comment:
you're missing the parameters that comes with this, that should come in with the params